### PR TITLE
modified azure/template.json accommodate multiple custom-domain (api\api2)

### PR DESCRIPTION
**Pull request by felixtheflex**
### Context

The change is to accommodate both api and ap2 custom domain requires for all environment.

### Changes proposed in this pull request

Ammending arm template to support multiple custom domains

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x ] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

File: azure/template.json [hunk 1]
```diff
@@ -35,6 +35,12 @@
         "description": "The custom hostname to add to the app service."
       }
     },
+    "customDomains": {
+      "type": "array",
+      "metadata": {
+        "description": "The custom hostname to add to the app service."
```
Comment (saliceti) [inline azure/template.json:41]: Should it describe custom domains instead?

File: azure/template.json [hunk 2]
```diff
@@ -501,8 +554,8 @@
               }
             ]
           },
-          "customHostName": {
-            "value": "[parameters('customHostName')]"
+          "customDomains": {
+            "value": "[reference('custom-domains').outputs.customDomains.value]"
           },
           "certificateThumbprint": {
             "value": "[if(greater(length(parameters('customHostname')), 0), reference('app-service-certificate', '2018-11-01').outputs.certificateThumbprint.value, '')]"
```
Comment (vigneshmsft) [inline azure/template.json:561]: this can be removed as well

File: azure/template.json (hunk 1 above)
Comment (misaka) [reply azure/template.json:41]: Actually, `api` and `api2` are the hostname segment, no? Should this be `fqdn`?

Comment (felixtheflex) [reply azure/template.json]: > 
> 
> Should it describe custom domains instead?

Descriptions now corrected......

**Pull request merged**